bin/tag-release.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
+set -euo pipefail
 
 moz_git_remote="${MOZ_GIT_REMOTE:-origin}"
+do_push=false
+
+# parse cli args
+while [[ $# -ge 1 ]]; do
+  key="$1"
+  case $key in
+    -P|--push)
```

Comment:
I don't mind the capital P (for the reason you stated) - just wanted to make sure it was intentional. 👍